products-service/src/utils/send-error.ts

```diff
@@ -0,0 +1,12 @@
+import { INTERNAL_SERVER_ERROR } from 'http-status-codes';
+
+import { AppError } from "../models/app-error";
+
+export const sendError = (error: AppError) => {
+  const { statusCode = INTERNAL_SERVER_ERROR } = error;
+
+  return {
+    statusCode,
+    body: JSON.stringify({ error })
```

Comment:
Seems like it's not a good idea to expose error stack trace due to security reasons. Could you please read this article from OWASP? https://owasp.org/www-community/Improper_Error_Handling

Reply:
Thank you for information! I've fixed this